model_files/S3/FlexibleSUSY.m.in

```diff
@@ -0,0 +1,64 @@
+
+FSModelName = "@CLASSNAME@";
+FSEigenstates = SARAH`EWSB;
+FSDefaultSARAHModel = S3;
+
+MINPAR = {{1, LambdaIN}};
+
+EXTPAR = {
+   {0, Qin},
+   {1, QEWSB},
+   {2, mS3IN},
+   {3, LambdaS3IN},
+   {4, LambdaHS3IN}
+};
+
+EWSBOutputParameters = { mu2 };
+
+HighScale = Qin;
+
+HighScaleFirstGuess = Qin;
+
+HighScaleInput = {{λ, LambdaIN}};
+
+SUSYScale = QEWSB;
+
+SUSYScaleFirstGuess = QEWSB;
+
+SUSYScaleInput = {{YS3L, LHInput[YS3L]}, {mS3, mS3IN}, {lambdaS3, LambdaS3IN}, {lambdaHS3, LambdaHS3IN}};
+
+LowScale = LowEnergyConstant[MZ];
+
+LowScaleFirstGuess = LowEnergyConstant[MZ];
+
+LowScaleInput = {
+   {v, 2 MZMSbar / Sqrt[GUTNormalization[g1]^2 g1^2 + g2^2]},
+   {Yu, Automatic},
+   {Yd, Automatic},
+   {Ye, Automatic}
+};
+
+InitialGuessAtLowScale = {
+   {v, LowEnergyConstant[vev]},
+   {Yu, Automatic},
+   {Yd, Automatic},
+   {Ye, Automatic}
+};
+
+ExtraSLHAOutputBlocks = {
+   {FlexibleSUSYOutput,
+           {{1, Hold[SUSYScale]},
+            {2, Hold[LowScale]} } },
+   {FlexibleSUSYLowEnergy, {
+         {20, FlexibleSUSYObservable`AMM[Fe[1]]},
+         {21, FlexibleSUSYObservable`AMM[Fe[2]]},
+         {26, FlexibleSUSYObservable`BrLToLGamma[Fe[2] -> {Fe[1], VP}]},
+         {27, FlexibleSUSYObservable`bsgamma}
+      }
+   }
+};
+
+SARAH`ParameterDefinitions = Append[
```

Comment:
I think this can be removed, because the model file seems to contain this description.

Reply:
Removed in 60049358013daf90eece343a9cb25ef2d6449b93.